### ISSUE TITLE
SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable: false positive yield $index++ => $item

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1174]
```php
<?php declare(strict_types = 1);

namespace Test;

use Generator;

class Bug {
    public function bug(): Generator {
        $index = 0;
        $items = [1,2,3];

        foreach ($items as $item) {
            yield $index++ => $item;
        }
    }
    public function ok(): Generator {
        $index = 0;
        $items = [1,2,3];

        foreach ($items as $item) {
            yield $index => $item;
        }
    }
}
```

```
FILE: /project/packages/Bug.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 13 | ERROR | Unused variable $index.
----------------------------------------------------------------------

Time: 775ms; Memory: 14MB
```

**Comment by kukulich** [comment]
@LastDragon-ru Have you tried `dev-master` ?

**Comment by LastDragon-ru** [comment]
@kukulich yep (a3e3726)

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/70cc5cdd984866d1dd70d5842222fa80117f3b24
